Libplanet.Tests/Net/Protocols/ProtocolTest.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Libplanet.Crypto;
+using Libplanet.Net;
+using Libplanet.Net.Protocols;
+using Serilog;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Libplanet.Tests.Net.Protocols
+{
+    public class ProtocolTest
+    {
+        private const int Timeout = 60 * 1000;
+        private readonly TestSwarmDictionary _dictionary;
```

Comment:
`TestSwarmDictionary` seems just `Dictionary<Address, TestSwarm>`'s wrapper. is it necessary?